### PR TITLE
[INLONG-5461][Sort] Add audit for MongoDB extract node

### LINKED ISSUE #5461: [Feature][Sort] Add audit for MongoDB extract node
### Description

Add Audit for mongoDB extract node

### Use case

User only need add with param "inlong.audit":"\<host\>&\<port\>" when using flink sql.
But user must set inlong.metric when set inlong.audit.
for example:
with param "inlong.metric":"\<groupId\>&\<streamId\>&\<nodeId\>"

example:
```sql
  create table `table_groupId_streamId_nodeId1`(
    `age` INT,
    `name` STRING,
    PRIMARY KEY(`age`) NOT ENFORCED)
    WITH (
   'connector' = 'mongodb-cdc-inlong',
    'hosts' = '9.135.81.137:27017',
    'database' = 'inlong',
    'collection' = 'inlong',
    'username' = 'flink',
    'password' = 'flinkpw',
   'inlong.audit' = 'host&port',
   'inlong.metric' = 'mongoGroup&mongoStream&mongoNode1'
);

CREATE TABLE `table_groupId_streamId_nodeId2`(
    PRIMARY KEY (`name`,`age`) NOT ENFORCED,
    `name` STRING,
    `age` INT)
    WITH (
    'connector' = 'print'
);

   INSERT INTO `table_groupId_streamId_nodeId2`
    SELECT
    `name` AS `name`,
    `age` AS `age`
    FROM `table_groupId_streamId_nodeId1`;
```
It will send audit data to [audit service](https://github.com/apache/inlong/tree/master/inlong-audit)

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gong**
### Prepare a Pull Request

- [INLONG-5461][Sort] Add Audit for mongoDB extract node

- Fixes #5461 

### Motivation

Add Audit for mongoDB extract node

### Modifications

* Add inlong.audit for mongoDB extract node
* Add audit computing for mongoDB extract node

### CODE REVIEW

File: inlong-sort/sort-connectors/mongodb-cdc/pom.xml [hunk 1]
```diff
@@ -41,6 +41,11 @@
             <artifactId>sort-connector-base</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.inlong</groupId>
+            <artifactId>audit-sdk</artifactId>
+            <version>${project.version}</version>
```
Comment (EMsnap) [inline inlong-sort/sort-connectors/mongodb-cdc/pom.xml:47]: already exist in connetor-base

Comment (gong) [reply inlong-sort/sort-connectors/mongodb-cdc/pom.xml:47]: I suggest write dependency directly. Because this module self use audit-sdk.

**Review by Oneal65 — APPROVED** [review]

File: inlong-sort/sort-connectors/mongodb-cdc/pom.xml (hunk 1 above)
Comment (yunqingmoswu) [reply inlong-sort/sort-connectors/mongodb-cdc/pom.xml:47]: Every connector will extends connector-base.

Comment (gong) [reply inlong-sort/sort-connectors/mongodb-cdc/pom.xml:47]: > Every connector will extends connector-base.

short path denpendcy is better

**Review by healchow — APPROVED** [review]

**Review by thesumery — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Review by yunqingmoswu — APPROVED** [review]
LGTM

**Pull request merged**